### ISSUE TITLE
QueryRow().Scan() does not work 

### ISSUE THREAD

**Issue by hqin** [SAP/cgo-ase#19]
**Description**

QueryRow().Scan()  fails for both queries without binding variables and with binding variables.
(go-ase driver fails for queries with binding variables in a separate bug report)

**Expected behaviour**
QueryRow().Scan() should return correct results.

**Current behaviour**
QueryRow().Scan() fails with the following error message:
```
received error reading results: Command failed: Routine failed
```

**Steps to Reproduce**
1. Export the following environment variables:
export DB_USERNAME=
export DB_PASSWORD=
export DB_DATABASE=
export DB_HOSTNAME=
export DB_PORT=
export DB_DRIVER=ase
export LD_LIBRARY_PATH=$SYBASE/OCS/lib:$SYBASE/OCS/lib3p:$SYBASE/OCS/lib3p64

2. Run test:
CGO_LDFLAGS="-L$SYBASE/OCS/lib"  go test -v

3. The errors are:
```
=== RUN   TestQueryRowWithoutBindingVars
    queryrow_test.go:25: querying failed: go-ase: received error reading results: Command failed: Routine failed
--- FAIL: TestQueryRowWithoutBindingVars (0.08s)
=== RUN   TestQueryRowWithBindingVars
    queryrow_test.go:41: querying failed: go-ase: received error reading results: Command failed: Routine failed
--- FAIL: TestQueryRowWithBindingVars (0.09s)
FAIL
```

**Environment**

Client:
  OS: Centos-7
  go version: 1.16.3
  cgo-ase version / commit hash: v0.0.0-20210312084010-d645a8f3e9d9
  go-dblib version / commit hash: v0.0.0-20210315102028-46975ce12d39

Server:
  OS: Centos-7 
  ASE version: 16.0_SP03PL02_EBF28306.el7

**Additional context**
[queryrow_test.go.gz](https://github.com/SAP/cgo-ase/files/6388008/queryrow_test.go.gz)

**Comment by hqin** [comment]
Forgot to mention the test data:
```
create table TEST_PROFILE(UserId BIGINT, ItemName varchar(32) not null, ItemValue varchar(32) not null)
create unique clustered index iuc_test_profile on TEST_PROFILE(UserId, ItemName)
insert into TEST_PROFILE values(1, "CUR_ACCOUNT", "1200")
```

**Comment by ntnn** [comment]
The error is fixed with #23
